module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,18 @@
+#include <algorithm>
+#include "vowel.hpp"
+
+void removeVowelsfromWord(std::string& word)
+{
+    for (char i : "aAeEyYuUiIiOo")
```

Comment:
I think you should make this "aAeEyYuUiIiOo" constrexpr and put it in the header to make it easier to modify in the future

Reply:
+1